dev_requirements.txt

```diff
@@ -15,6 +15,7 @@ flake8==6.0.0
 black==22.10
 mypy==1.0.0
 isort==5.12.0
+types-tabulate
```

Comment:
Good question! `types-tabulate` is a "stub" package that provides types for all the functions/variables in the `tabulate` package since `tabulate` itself does not provide type info out of the box. It is needed for mypy to run.